Source/effects.h

```diff
@@ -742,120 +742,348 @@ enum _sfx_id : int16_t {
 	PS_ROGUE100,
 	PS_ROGUE101,
 	PS_ROGUE102,
+	// CHAMBER OF BONE intro: Beyond the hall of heroes...
 	PS_WARR1,
+
+	// Then by binding spirit to the crafted rock. This sounds like a spell of some kind.
 	PS_WARR2,
+
+	// Let this tome serve as your warning... this is a forbidden place. return from whence you came and your life will be spar1ed
 	PS_WARR3,
+
+	// The creature of flame, by horazon. Should be interesting reading.
 	PS_WARR4,
+
+	// Morta vespa gaia ilumino...
 	PS_WARR5,
+
+	// The liutenant of belial is known as grimspike

```

Comment:
The lieutenant

Reply:
don't bother, it's a draft for a reason, a lot of typos etc. : P